PacketDefinitions420/PacketNotifier.cs

```diff
@@ -87,7 +88,7 @@ public void NotifyPing(ClientInfo client, Vector2 pos, int targetNetId, Pings ty
             _packetHandlerManager.BroadcastPacketTeam(client.Team, response, Channel.CHL_S2C);
         }
 
-        public void NotifyTint(TeamId team, bool enable, float speed, Color color)
+        public void NotifyTint(TeamId team, bool enable, float speed, GameServerCore.Content.Color color)
```

Comment:
Add a ``using`` directive.

Reply:
I find this ambiguity itself to be an issue. It's a separate issue, but maybe the two should either be unified or one of them renamed.

Reply:
I feel it may be easier in general if we just added LeaguePackets project and edit from there.

Reply:
I feel it may be easier in general if we just added LeaguePackets project and edit from there.